provisioning/tasks/dashboard.yml

```diff
@@ -0,0 +1,11 @@
+- name: Ensure the dashboard directory exists
+  file:
+    path: "{{ dashboard_install_dir }}"
+    state: directory
+    mode: 0755
```

Comment:
Sure, I would also need to add a check to the drush alias templates. That's no biggy though.